test/test_shared_library.py

```diff
@@ -0,0 +1,98 @@
+    test_icone_data=translator_shared_library.parse_xml(test_input)
+    print(json.dumps(test_icone_data))
+    assert test_icone_data==valid_icone_data
+
+#--------------------------------------------------------------------------------unit test for parse_arguments function--------------------------------------------------------------------------------
+def test_parse_arguments():
+    test_input='-i inputfile.xml -o outputfile.geojson'
+    test_argv=test_input.split(' ')
+    test_input,test_output=translator_shared_library.parse_arguments(test_argv)
+    assert test_output== test_argv[3] and test_input==test_argv[1]
+
+def test_parse_arguments_no_arguments():
+    test_input= ''
+    test_argv=test_input.split(' ')
+    test_input,test_output=translator_shared_library.parse_arguments(test_argv)
+    assert test_output== 'wzdx_translated_output_message.geojson' and test_input== ''
+
+def test_parse_arguments_invalid_arguments():
+    test_input= 'inputfile.xml outputfile.geojson'
+    test_argv=test_input.split(' ')
+    test_input,test_output=translator_shared_library.parse_arguments(test_argv)
+    assert test_output== 'wzdx_translated_output_message.geojson' and test_input== ''
+
+def test_parse_arguments_help():
+    test_input= '-h'
+    test_argv=test_input.split(' ')
+    with pytest.raises(SystemExit):
+        translator_shared_library.parse_arguments(test_argv)
+
+#--------------------------------------------------------------------------------unit test for validate_write function--------------------------------------------------------------------------------
+def test_validate_wzdx_valid_wzdx_data():
```

Comment:
Can we add a few more cases here, for instance what happens if the schema is empty, or the json file is?

Reply:
i added some edge cases